packages/client/src/app/resolvers/work-page.resolver.ts

```diff
@@ -8,6 +8,9 @@ import { WorkPageData } from '../models/site';
 import { AuthService } from '../services/auth';
 import { HistoryService, LocalSectionsService, WorksService } from '../services/content';
 
+
+// This resolver is obsolete. 
+// TODO: Delete.
 @Injectable()
 export class WorkPageResolver implements Resolve<WorkPageData> {
```

Comment:
I'm a little afraid to remove this outright, so just a TODO for now. We can go through and do a removal pass once the dust from the API change clears.